src/ProjectSystem/Targets/Microsoft.NET.Sdk.Web.ProjectSystem.props

```diff
@@ -28,25 +28,26 @@ Copyright (c) .NET Foundation. All rights reserved.
 
   <ItemGroup Condition=" '$(EnableDefaultItems)' == 'true' And '$(EnableDefaultContentItems)' == 'true' ">
     <!-- Publish everything under wwwroot, all JSON files, all config files and all Razor files -->
-    <Content Include="wwwroot\**" CopyToPublishDirectory="PreserveNewest" Exclude="$(DefaultItemExcludes);$(DefaultExcludesInProjectFolder)" />
-    <Content Include="**\*.config" CopyToOutputDirectory="PreserveNewest" CopyToPublishDirectory="PreserveNewest" Exclude="$(DefaultItemExcludes);$(DefaultExcludesInProjectFolder);$(DefaultWebContentItemExcludes)" />
-    <Content Include="**\*.json" CopyToOutputDirectory="PreserveNewest" CopyToPublishDirectory="PreserveNewest" Exclude="$(DefaultItemExcludes);$(DefaultExcludesInProjectFolder);$(DefaultWebContentItemExcludes)" />
+    <Content Condition=" '$(DisableWebSdkContent)' == '' " Include="wwwroot\**" CopyToPublishDirectory="PreserveNewest" Exclude="$(DefaultItemExcludes);$(DefaultExcludesInProjectFolder)" />
+    <Content Condition=" '$(DisableWebSdkContent)' == '' " Include="**\*.config" CopyToOutputDirectory="PreserveNewest" CopyToPublishDirectory="PreserveNewest" Exclude="$(DefaultItemExcludes);$(DefaultExcludesInProjectFolder);$(DefaultWebContentItemExcludes)" />
+    <Content Condition=" '$(DisableWebSdkContent)' == '' " Include="**\*.json" CopyToOutputDirectory="PreserveNewest" CopyToPublishDirectory="PreserveNewest" Exclude="$(DefaultItemExcludes);$(DefaultExcludesInProjectFolder);$(DefaultWebContentItemExcludes)" />
 
     <!-- Set CopyToPublishDirectory to Never for items under AppDesignerFolder ("Properties", by default) to avoid publishing launchSettings.json -->
     <Content Update="$(AppDesignerFolder)\**" CopyToPublishDirectory="Never" Condition="'$(AppDesignerFolder)' != ''"/>
-    
+
     <!-- Remove Content items from other item types (in a way that CPS understands) -->
-    <None Remove="wwwroot\**;**\*.json;**\*.config" />
-    <Compile Remove="wwwroot\**" />
-    <EmbeddedResource Remove="wwwroot\**" />
-    
+    <None Condition=" '$(DisableWebSdkContent)' == '' " Remove="wwwroot\**" />
+    <None Remove="**\*.json;**\*.config" />
```

Comment:
Why wasn't this added to the Razor SDK? The `Content` Include is part of the Razor SDK, think this should be copied too to complement it.

Reply:
@pranavkm This includes appSettings.json and similar, but we can move it too.

Reply:
I was trying to keep the things I moved to a minimum, but it might be better for it to be there too. I'll move it

Reply:
Agreed. Since we are moving the globs, we should move all the globs.

Reply:
Agreed, all globs should move.

Note that this condition is odd
`Condition=" '$(DisableWebSdkContent)' == '' "`

Setting `<DisableWebSdkContent>false</DisableWebSdkContent>` will still disable it.  I think it should be authored `Condition=" '$(DisableWebSdkContent)' != ''true'"`

Reply:
Done